tests/48admin.pl

```diff
@@ -0,0 +1,31 @@
+test "/whois",
+   requires => [ $main::API_CLIENTS[0] ],
+
+   do => sub {
+      my ( $http ) = @_;
+
+      my $user;
+
+      matrix_register_user( $http )
+      ->then( sub {
+         ( $user ) = @_;
+
+         do_request_json_for( $user,
+            method => "GET",
+            uri    => "/api/v1/admin/whois/".$user->user_id,
+         )
+      })->then( sub {
+         my ( $body ) = @_;
+
+         assert_json_keys( $body, "devices", "user_id" );
```

Comment:
For consistency with the other tests this wants to be

    assert_json_keys( $body, qw( devices user_id ));